src/site/content/en/blog/building-a-tooltip-component/index.md

```diff
@@ -0,0 +1,717 @@
+---
+layout: post
+title: Building a tooltip component
+subhead: A foundational overview of how to build a color-adaptive and accessible tooltip custom element.
+authors:
+  - adamargyle
+description: A foundational overview of how to build a color-adaptive and accessible tooltip custom element.
+date: 2022-10-24
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/Mx5dDmELgbJottijzfwF.png
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/5XgqLJdeI33MDjvJu2te.png
+alt: A tilted grid of hotpink FABs.
```

Comment:
I don't think this alt relates to the image in use.

Reply:
nice catch, def a copy paste issue! i'll fix 🙂